src/Profiles/Shifting/ANTPLUS_ShiftingPrivateDefines.h

```diff
@@ -0,0 +1,23 @@
+#ifndef ANTPLUS_SHIFTINGPROFILEPRIVATEDEFINES_h
+#define ANTPLUS_SHIFTINGPROFILEPRIVATEDEFINES_h
+
+/* Channel Config */
+#define ANTPLUS_SHIFTING_CHANNELTYPE CHANNEL_TYPE_BIDIRECTIONAL_RECEIVE 
```

Comment:
Add side of the profile to define `ANTPLUS_SHIFTING_DISPLAY_CHANNELTYPE`

Reply:
Changed